network/eni/eni.go

```diff
@@ -102,6 +97,27 @@ func (eni *ENI) AttachToLink() error {
 	return nil
 }
 
+// getInterfaceByMac gets the interface with specified mac address, and picks the one with shortest name
+// if there are multiple matches, which can happen temporarily when setting up a branch ENI, in which it
+// can have the same mac address as the trunk during branch.AttachToLink.
+func getInterfaceByMac(macAddress net.HardwareAddr, interfaces []net.Interface) *net.Interface {
```

Comment:
Can you please call this getInterfaceByMACAddress, to comply with the Go naming conventions? Also please move to the end of the file to keep the ENI class methods together.

Reply:
updated